### PR TITLE
Solana projects

### PR DESCRIPTION
**Pull request by rmshea**
Hey Electric Capital team,

I added a few repositories of note here. Thanks for keeping this updated!

Ryan

### CODE REVIEW

**Review by jubos — CHANGES_REQUESTED** [review]
Thanks for the PR, mind rebasing to master as I see a few duplicates in here.

**Comment by rmshea** [comment]
hey @jubos sorry for the delay, just rebased.

File: data/ecosystems/s/solana.toml [hunk 1]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
 [[repo]]
 url = "https://github.com/zjhmale/solana-explorer-tui"
 
+[[repo]]
+url = "https://github.com/Nebula-Wallet"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:572]: This is a github org instead of a repo.

File: data/ecosystems/s/solana.toml [hunk 2]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/solongwallet/solongext"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:575]: 404 repo.

File: data/ecosystems/s/solana.toml [hunk 3]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/civicteam/identity-swap"
+
+[[repo]]
+url = "https://github.com/Xuriwork/spl-token-wallet"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:581]: 404 repo.

File: data/ecosystems/s/solana.toml [hunk 4]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
 [[repo]]
 url = "https://gitlab.com/sigrlami/omuon"
+
+[[repo]]
+url = "https://github.com/ErJavid/solhackathondapp"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:587]: dup

File: data/ecosystems/s/solana.toml [hunk 5]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/decefi/decefi-solana"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:590]: 404 repo

File: data/ecosystems/s/solana.toml [hunk 6]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/haggle-quest/solana-game"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:593]: 404 repo

File: data/ecosystems/s/solana.toml [hunk 7]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/genesis-lab-team/solana-overview"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:596]: 404

File: data/ecosystems/s/solana.toml [hunk 8]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/paul-schaaf/diy_etf"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:599]: dup

File: data/ecosystems/s/solana.toml [hunk 9]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/distributedtown/distributed-town-backend"
+
+[[repo]]
+url = "https://github.com/mehmeta/spl-token-wallet"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:605]: 404

File: data/ecosystems/s/solana.toml [hunk 10]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/TitansFinance/apis-core/"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:608]: 404

File: data/ecosystems/s/solana.toml [hunk 11]
```diff
@@ -571,5 +568,44 @@ url = "https://github.com/Zergity/static"
...
+
+[[repo]]
+url = "https://github.com/anshuman73/contright"
```
Comment (jubos) [inline data/ecosystems/s/solana.toml:611]: 404

**Review by jubos — CHANGES_REQUESTED** [review]
I think I might have tried to pull from the same list as you (Solana hackathon airtable) at some point but a lot of these repos 404 at this point.

**Pull request merged**